rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,94 @@
+
+As a consumer of HCA data, I will be more confident in using raw data in my own analysis if they have been successfully run on the DCP-hosted pipelines.
+
+As a consumer of HCA data, I will be more confident in using metadata, raw data, or analysis products from an HCA project if all data and data provenance are available and easily referenced.
+
+As a consumer of HCA data, I do not want to see HCA Integration Test projects on the DataBrowser or DataStore, especially if I have a notification set up for new HCA projects as these tests will trigger my notification but don’t actually represent new data.
+
+As a member of the DCP, I cannot do a comprehensive final check on data on the DataBrowser or DataStore unless I see it as the data consumer will see it.
+
+As a member of the DCP, I do not want inaccurate metadata or products of incorrect data analysis being shown to the public. 
+Examples of inaccuracies that were/are publicly accessible:
+- Optimus results on Smart-seq2 data ([#2314](https://github.com/HumanCellAtlas/data-store/issues/2314))
+  - Impact: Incorrect analysis results are accessible to the user. Resulted in breaking data permanency as the data were removed with no record of provenance.
+- ‘Homo sapiens’ vs ‘homo sapiens’ ([ZD ticket 158](https://humancellatlas.zendesk.com/agent/tickets/158))
+  - Impact: Inconsistent access experience across HCA projects (a user specifying one value will get results, but not all desired results).
+- HCA Project ingested at the wrong schema version ([#2421](https://github.com/HumanCellAtlas/data-store/issues/2421))
+  - Impact: Inconsistent access experience across HCA projects. Resulted in breaking data permanency as the data were removed and resubmitted with new uuids and HCA project page url with no record of provenance.
+- HCA Project page Metadata tsv download threw an error
+  - Impact: A non-functioning user-accessible link. Inconsistent access experience across HCA projects if the metadata tsv is not available for all HCA projects.
+- HCA submitter name misspelled ("Tom Mithcell")
+  - Impact: A (rather personal) HCA submitter-facing error will put DCP credibility at risk.
+
+As an HCA submitter whose name is associated with the HCA project, I want to be allowed time to review the full details of my HCA submission as for accuracy before it becomes publicly available. I want this review to be through the vantage of a potential user. (possibly 2nd phase)
+
+As an HCA submitter, I am curious where my HCA submission is in the DCP’s data lifecycle after I have provided the metadata spreadsheet all the way until it is released. (possibly 2nd phase)
+
+
+## Functional Specifications
+**Phase 1**
+*MVP requirements*
+- A mechanism for DCP members to sign-in to the DataBrowser and DataStore.
+- A mechanism for an HCA project to be visible only for a user that is signed-in.
```

Comment:
The portal is not the only access point in the system. I believe this would require authN/Z to all API endpoints and for those endpoints to censor data based on the state of the HCA project.

Reply:
I think we're aligned in our ideas here. A likely next step if/when the RFC is accepted will be a focused call (or F2F session) with all the right people present and figure out implementation.